src/shared/base-filesystem.c

```diff
@@ -21,26 +21,34 @@
 #include "umask-util.h"
 #include "user-util.h"
 
+typedef enum BaseFilesystemModifier {
```

Comment:
nah, I prefer the previous version with flags. The _IGNORE_FAILURE one is independent of others, i.e. it is only a minor detail/coincidence that only one flag is used for each entry currently.

Reply:
Ok, I liked the enum a bit better, but not too important either way